genie-agent/src/main/java/com/netflix/genie/agent/cli/ExitCode.java

```diff
@@ -0,0 +1,43 @@
+/*
+ *
+ *  Copyright 2018 Netflix, Inc.
+ *
+ *     Licensed under the Apache License, Version 2.0 (the "License");
+ *     you may not use this file except in compliance with the License.
+ *     You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *     Unless required by applicable law or agreed to in writing, software
+ *     distributed under the License is distributed on an "AS IS" BASIS,
+ *     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *     See the License for the specific language governing permissions and
+ *     limitations under the License.
+ *
+ */
+
+package com.netflix.genie.agent.cli;
+
+/**
+ * Exit codes for Genie agent.
+ *
+ * @author mprimi
+ * @since 4.0.0
+ */
+enum ExitCode {
```

Comment:
Could this want some string message along with them that we could include with the "HELP" or other?